### DISCUSSION TITLE
Get palette colors

### DISCUSSION THREAD

**Discussion by eeeXun** [ellisonleao/gruvbox.nvim#278] [General]
Before #276, I get the gruvbox color with

```lua
local colors = require("gruvbox.palette").colors

local bright_red = colors.bright_red
...
local faded_red = colors.faded_red
...
local neutral_red = colors.neutral_red
...
```

But after the #276, it seems that I have to change my code to

```lua
local colors_dark = require("gruvbox.palette").get_base_colors({}, "dark", nil)
local colors_light = require("gruvbox.palette").get_base_colors({}, "light", nil)

local bright_red = colors_dark.red
...
local faded_red = colors_light.red
...
local neutral_red = colors_dark.red
```

Also, after #276, there is no way to get `neutral_orange`.
So can we re-expose `require("gruvbox.palette").colors`? Or is there any solutioin?

**Comment by ellisonleao** [comment]
hey @eeeXun i am working on a major module rewrite which will expose the base palette colors. I will close this one here when it's done

**Comment by ellisonleao** [comment]
PR Opened https://github.com/ellisonleao/gruvbox.nvim/pull/280
